client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/EndpointListDecoder.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.client.armeria;
+
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import com.linecorp.armeria.client.Endpoint;
+
+/**
+ * Decode and encode between CentralDogma upstream file content and list of {@link Endpoint}s.
```

Comment:
`Decodes the content of a file in Central Dogma into a list of {@link Endpoint}s.`?